code.js

```diff
@@ -68,33 +11,28 @@ const listOfBooksElement = document.querySelector('#list-books .list-of-books');
 function deleteBookElement(parentContainer, id) {
   parentContainer.remove();
   library.remove(id);
-  updateStoreFormData(library);
 }
 
 function CreateBookItemHTML(id, title, author) {
   const divContainer = document.createElement('div');
-  const bookTitleElement = document.createElement('p');
-  const bookAuthorElement = document.createElement('p');
+  const bookTitleAndAuthorElement = document.createElement('p');
   const deleteBookBtn = document.createElement('button');
-  const hrElement = document.createElement('hr');
 
   divContainer.id = `Book-${id}`;
+  divContainer.classList.add('books-details');
 
```

Comment:
- [ ] I have seen that you have refactored the code in separate files which is good 👏 (i.e: in the js folder) and you kept the initial code in the `code.js` file and it is still referenced in your `index.html` file **while it contains the code with functions while we have to use classes**. However, I realized that the application is not working. I can't add the book and therefore I can't remove any. Let me show you the error being displayed in the browser console 👇 
![image](https://user-images.githubusercontent.com/90222110/182466951-b7128c61-faa4-46d2-a344-152f239b3fb5.png)

**PLEASE reference your entry point js file that uses classes in your `index.html` and see if the application works fine. However, you can still work to make the application perform as expected**

Reply:
fixed, this happens because local storage initialisation was not null safe.